### PR TITLE
add debug log so we know when it converges

### PR DESCRIPTION
**Pull request by honnix**
@danielnorberg Does this make sense?

### CODE REVIEW

File: styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java [hunk 1]
```diff
@@ -267,6 +267,9 @@ void concurrentlyIndexActiveWorkflowInstances() {
               return null;
             }
 
+            LOG.debug("adding the missing entry {} to {}", expectedEntry,
```
Comment (danielnorberg) [inline styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java]: Are we running with debug logging enabled?

Perhaps the log message should mention workflow instances and index?

I'd be tempted to track this using a metric but I guess we won't be keeping this around for more than a week or so.

**Comment by danielnorberg** [comment]
👍 except for comments

File: styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java (hunk 1 above)
Comment (honnix) [reply styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java]: Yeah we do now. Will add more info to the message. Didn’t want to bother metrics as you said.

**Comment by honnix** [comment]
We can all remove the syncing code I believe. Nothing has been observed.

**Pull request merged**